src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    private String description;
+    private boolean isDone;
+
+    Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    void markAsDone() {
+        isDone = true;
+    }
+
+    String saveFormat() {
+        return String.format("%s|%s", this.description, this.isDone);
+    }
+
+    private String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Very clean and good usage of a ternary operator here as opposed to an if else statement!